src/Sarif.Converters/SemmleQlConverter.cs

```diff
@@ -14,9 +14,9 @@ namespace Microsoft.CodeAnalysis.Sarif.Converters
     /// <summary>
     /// Converts a log file from the Semmle format to the SARIF format.
     /// </summary>
-    internal class SemmleQLConverter : ToolFileConverterBase
+    public class SemmleQLConverter : ToolFileConverterBase
```

Comment:
>public [](start = 4, length = 6)

At least some of the other converters (AndroidStudio, FortifyFpr ... I didn't look at them all) are `internal`. Why did you decide to make this one `public`?